src/modules/phonenum/Makefile

```diff
@@ -5,13 +5,8 @@ include ../../Makefile.defs
 auto_gen=
 NAME=phonenum.so
 
-CXX=g++
-LD=g++
-LIB_DIR = /opt/local/lib
-
-LIBS= -L${LIB_DIR} cphonenumber.o -lphonenumber -lgeocoding
-
```

Comment:
The cphonenumber.o is needed for linking, not sure it would work without, expecting missing symbols. Have you started kamailio with this module, or just compiled it?

Reply:
Yep, you are right.